app/views/users/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<p id="notice"><%= notice %></p>
+<div class="groupon_signup_form">
+<%= form_for(user) do |f| %>
+  <legend><%= t('.signup_form_legend') %></legend>
+  <div class="form-container">
+  <div class="field">
+    <%= f.label :name , class: 'col-form-label' %>
+    <%= f.text_field :name, size: 40, class: 'form-control', placeholder: 'Enter name' %>
```

Comment:
remove size from here, we normally use css to handle width, you can use bootstarp grid structure for the same